templates/federated_reporting/mrproper.py

```diff
@@ -0,0 +1,133 @@
+#!/bin/python3
+import subprocess
+import urllib3
+import json
+import platform
+
+MRPROPER_DIR = "/vagrant"
```

Comment:
@nickanderson where would you place these bits? self-signed certificates from superhub, feeders, cfsecret files with superhub and feeder mrproper account credentials?

Reply:
maybe `/opt/cfengine/federation/mrproper` !?